components/builder-originsrv/src/migrations/origin_channels.rs

```diff
@@ -130,9 +130,7 @@ pub fn migrate(migrator: &mut Migrator) -> Result<()> {
                           WHERE o.name = op_origin
                           AND oc.name = op_channel
                           AND op.ident LIKE (op_ident  || '%')
-                          AND op.target = op_target
-                          ORDER BY op.ident DESC
-                          LIMIT 1;
+                          AND op.target = op_target;
```

Comment:
Will we need to manually run this function update on live or did we do this the other night/morning?